### PR TITLE
test(ComboBox): add vrt

### LINKED ISSUE #9268: Add VRT for ComboBox
This issue is to implement visual regression testing (VRT) for the following components by adding baseline Percy snapshot(s) inside a `*-test.e2e.js` file. More info and high level steps to complete can be found in the umbrella issue, #8911 

- [ ] ComboBox

### PR DESCRIPTION
**Pull request by jnm2377**
Closes #9268 

Adds percy snapshots for ComboBox

#### Testing / Reviewing

- view Percy test details, check ComboBox snapshot, ensure styles are applied correctly, approve snapshot if visuals are correct :) 
- ensure all tests are passing

### CODE REVIEW

**Comment by jnm2377** [comment]
@tay1orjones I tried to add a click event on this like Accordion tests, and check to see that the list items are showing, but I'm running into issues with it. I get an error when I try to check for `Option 1` text. Any idea what I'm doing wrong?

**Comment by jnm2377** [comment]
Ok, I think I fixed the error I was running into. Should be ready for review.

**Review by tay1orjones — APPROVED** [review]

File: packages/react/src/components/ComboBox/ComboBox-test.e2e.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+import ComboBox from './ComboBox';
+
+describe('ComboBox', () => {
+  beforeEach(() => {
+    const items = [
+      {
+        id: 'option-0',
+        text:
+          'An example option that is really long to show what should be done to handle long text',
+      },
```
Comment (tay1orjones) [inline packages/react/src/components/ComboBox/ComboBox-test.e2e.js:23]: Forgot to mention something I found useful for Dropdown was the `generateItems` and `generateGenericItem` helpers from `../ListBox/test-helpers`. It helps cut down the boilerplate a bit

**Review by tw15egan — APPROVED** [review]
Sorry for the late review! Nice work

**Pull request merged**